config/v1/types_infrastructure.go

```diff
@@ -478,6 +478,32 @@ type AzurePlatformStatus struct {
 	// armEndpoint specifies a URL to use for resource management in non-soverign clouds such as Azure Stack.
 	// +optional
 	ARMEndpoint string `json:"armEndpoint,omitempty"`
+
+	// resourceTags is a list of additional tags to apply to Azure resources created for the cluster.
+	// See https://docs.microsoft.com/en-us/rest/api/resources/tags for information on tagging Azure resources.
+	// Azure supports a maximum of 50 tags per resource except for few, which have limitation of 15 tags.
+	// OpenShift reserves 5 tags for its internal use, and allows 10 tags for user configuration.
+	// +kubebuilder:validation:MaxItems=10
+	// +optional
+	ResourceTags []AzureResourceTag `json:"resourceTags,omitempty"`
+}
+
+// AzureResourceTag is a tag to apply to Azure resources created for the cluster.
+type AzureResourceTag struct {
+	// key is the key of the tag
```

Comment:
Please explain in the godoc what the validation patter and maximum length are, these aren't converted into godoc from the kubebuilder tags

Reply:
comment incorporated.